docs/2_step_by_step/2_4_create_training_data.md

```diff
@@ -1 +1,102 @@
-# Create training data
\ No newline at end of file
+# Create training and evaluation (Optional) data
+
+Finetuner accepts [docarray](https://docarray.jina.ai/) `DocumentArray` as training data and evaluation data.
+Because Finetuner follows a [supervised-learning](https://en.wikipedia.org/wiki/Supervised_learning) scheme,
+you should assign label to each `Document` inside `DocumentArray` with:
+
+````{tab} text-to-text search
+```python
+from docarray import Document, DocumentArray
+
+train_da = DocumentArray([
+    Document(
+        content='pencil skirt slim fit available for sell',
+        tags={'finetuner_label': 'skirt'}
+    ),
+    Document(
+        content='stripped over-sized shirt for sell',
+        tags={'finetuner_label': 't-shirt'}
+    ),
+    ...,
+])
+```
+````
+````{tab} image-to-image search
+```python
+from docarray import Document, DocumentArray
+
+train_da = DocumentArray([
+    Document(
+        content='https://...skirt-1.png',
+        tags={'finetuner_label': 'skirt'},
+    ),
+    Document(
+        content='https://...t-shirt-1.png',
+        tags={'finetuner_label': 't-shirt'},
+    ),
+    ...,
+])
+```
+````
+````{tab} text-to-image search on CLIP
+```python
+from docarray import Document, DocumentArray
+
+train_da = DocumentArray([
+    Document(
+        chunks=[
+            Document(
+                content='pencil skirt slim fit available for sell',
+                modality='text',
+                tags={'finetuner_label': 'skirt-1'}
+            ),
+            Document(
+                content='https://...skirt-1.png',
+                modality='image',
+                tags={'finetuner_label': 'skirt-1'}
+            ),
+        ],
+    ),
+    Document(
+        chunks=[
+            Document(
+                content='stripped over-sized shirt for sell',
+                modality='text',
+                tags={'finetuner_label': 'shirt-1'}
+            ),
+            Document(
+                content='https://...shirt-1.png',
+                modality='image',
+                tags={'finetuner_label': 'shirt-1'}
+            ),
+        ],
+    ),
+])
+```
+````
+
+As shown in the above code blocks.
+For single-modality model fine-tuning, you only need to create a `Document` with `content` and `tags` with the `finetuner_label`.
+
+For cross-modality (text-to-image) fine-tuning with CLIP,
+you should create a root `Document` which wraps two `chunks` with the `image` and `text` modality.
```

Comment:
will we use `dataclass` here? Or just two different `chunk`s?

Reply:
it's tricky, you can not assign modality in dataclass and you can not assign label